docs/hopsworksai/aws/restrictive_permissions.md

```diff
@@ -226,7 +226,6 @@ For this remove the following permissions from the instance profile:
       "S3:PutLifecycleConfiguration", 
       "S3:GetLifecycleConfiguration", 
       "S3:PutBucketVersioning", 
-      "S3:GetBucketVersioning"
```

Comment:
I'm surprised that it is the `GetBucketVersioning` and not the `PutBucketVersioning` that we remove. Why de we need to be able to put versioning?

Reply:
the paragraph above talks about removing permissions that are not needed. now we need GetBucketVersioning for both enterprise and non-enterprise versions.

Reply:
Ok, I went too fast and thought it was the same as for azure.